contracts/payloads/IGP80/PayloadIGP80.sol

```diff
@@ -368,6 +376,36 @@ contract PayloadIGP80 is PayloadIGPConstants, PayloadIGPHelpers {
         FLUID_RESERVE.approve(protocols, tokens, amounts);
     }
 
+    // @notice Action 10: Update USDe-USDT DEX Lower Range and Fee %
```

Comment:
update is for both lower and upper range, we can simply write Range 

also on comment below

Reply:
correct my bad updating